server/middleware/insertJtiIntoDB.js

```diff
@@ -0,0 +1,23 @@
+const { JwtIds } = require('../../database/models');
+const { v4: uuidv4 } = require('uuid');
+const logger = require('../../config/logger');
+
+const insertJtiIntoDB = (request, response, next) => {
+	const { userId } = request.body;
+	const jti = uuidv4();
+	const TEN_MINS_IN_SECONDS = 10 * 60;
+	const expiryTime = Math.floor(Date.now() / 1000) + TEN_MINS_IN_SECONDS;
```

Comment:
Oh I see. Is there a reason why we're doing this complicated acrobatics that I'm missing?